api/v2/models/bouquets/bouquets_model.py

```diff
@@ -40,8 +38,11 @@ def receive_notification_from_google(self):
 
         # TODO: add functionality to receive calendar notifications for purposes of logging them on the relay log table
 
-    def add_bouquet(self, bouquet_id):
+    @staticmethod
+    def add_bouquet(**kwargs):
         """Method for adding a bouquet"""
+        x = Bouquets(**kwargs)
```

Comment:
this variable X isn't descriptive at all. if not mistaken though

Reply:
Right, let me change this asap